roles/openstack-setup/tasks/networks.yml

```diff
@@ -1,16 +1,18 @@
 ---
 - name: neutron networks
-  neutron_network: name={{ item.name }}
-                   provider_network_type={{ item.network_type }}
-                   state=present
-                   shared={{ item.shared }}
-                   router_external={{ item.external }}
-                   provider_segmentation_id={{ item.segmentation_id }}
-                   provider_physical_network={{ item.provider_physical_network }}
-                   auth_url={{ endpoints.auth_uri }}
-                   login_tenant_name=admin
-                   login_username=admin
-                   login_password={{ secrets.admin_password }}
+  os_network:
```

Comment:
There have been some updates to the os_network module recently upstream, which included support for `local` network type, which we're using here. The version upstream in 2.1.x does not support that, so we may need to vendor in the latest upstream version of `os_network`.